gateway/src/services/wallet/wallet.ts

```diff
@@ -0,0 +1,11 @@
+// import { ConfigManagerCertPassphrase } from '../config-manager-cert-passphrase';
```

Comment:
Looks like this file has been superseded by `wallet.controller.ts`? Let's just delete this file?